traits/ctraits.c

```diff
@@ -171,9 +171,6 @@ static int call_notifiers ( PyListObject *, PyListObject *,
 /* The default_value of the trait is the default value */
 #define CONSTANT_DEFAULT_VALUE 0
 
-/* The default_value of the trait is Missing */
-#define MISSING_DEFAULT_VALUE 1
```

Comment:
Possibly add a comment about why 1 is no longer used?

Reply:
I'd prefer to leave the history in the version control history, where it belongs. That is, I definitely don't want to leave explanations about *why* 1 is no longer used here, but it may make sense just to note the fact that 1 is no longer used, without going into why.